policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_us.model_api import *
+
+
+class az_family_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Family Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.az.tax.income.credits.family_tax_credits
+        income = tax_unit("adjusted_gross_income", period)
```

Comment:
we need to use arizona agi and not federal agi -> please double check in tax form and law